### ISSUE TITLE
Remove Lalasa's writing

### ISSUE THREAD

**Issue by EirinYagokoro** [taisei-project/taisei#179]
Please remove Lalasa's writing. Since you seem to be ignoring her at this point, it would only make sense for you to remove her text and replace it with anything else for the time being. The fact you're still keeping her text while trying to gather money to invest in the project comes off across as incredibly unethical and could cause you problems down the line.

**Comment by FookinAkari69** [comment]
I agree with this individual, this is a shameful display of a developer.

**Comment by Akaricchi** [comment]
Hi.

First of all, can you please tell us where the assumption that we are "ignoring" her comes from? As far as I'm aware, she (unfortunately) has left the project of her own accord, has not expressed a desire to withdraw her contributions, and has never tried to contact us (or at the very least, me) since. I believe she has actually given @laochailan an explicit permission to keep the writing in a private conversation, but I'll leave that to him to confirm that (I may be wrong here).

I would, in fact, gladly welcome her back on the team if she wants to. And I also have no problem whatsoever with withdrawing her writing, but only if she actually wants that, and comes here personally to express that.

**Comment by Lyricana** [comment]
Hello, it's me, Lalasa, for real.  You guys should know this account of mine too.
I did give laochailan explicit permission (even though I was going to remove it the first time around) because I felt guilty pulling it out when it had no suitable replacement.  I have been actually hoping all this time that it would be replaced by someone else's and I could leave this peacefully.  However, this is not the case and I see you've made a patreon where, for whatever reason, it's brought up that there is a writer on the team.  Now, I understand that there's barely any money being made and honestly I don't even want it, but if you are going to go this route, then I don't want my work to be monetized in this way.  I know I will be unlikely to see any of it, and now it's embarrassing me.  I did this under the expectation that this game would remain free and no profit would be made off of it.  Technically speaking it could be argued that this isn't profit but donating, but at the very least I should not be mentioned on your Patreon at all because I feel it is dishonest.  So please, if you're going to be garnering Patreon support like this, you should at least find a writer to replace my work and pay them for it.

**Comment by Akaricchi** [comment]
Hello @Lyricana, it's nice to see you again in spite of the circumstances.

>I did give laochailan explicit permission (even though I was going to remove it the first time around) because I felt guilty pulling it out when it had no suitable replacement. I have been actually hoping all this time that it would be replaced by someone else's and I could leave this peacefully.

That's very unfortunate. I honestly don't really understand why you'd want this, and I still have some hope that we can reconcile and work together again. But if that's not what you want, then I can't help it and won't insist.

>Now, I understand that there's barely any money being made and honestly I don't even want it, but if you are going to go this route, then I don't want my work to be monetized in this way. I know I will be unlikely to see any of it, and now it's embarrassing me.

Well, I would actually be totally fine with giving you a cut of it, or straight up (retroactively now) paying for your services, but you never asked for it, and now you just said you don't even want it (feel free to reconsider). The "I will be unlikely to see any of it" statement is not true, and I don't think you should be embarrassed by anything here.

>However, this is not the case and I see you've made a patreon where, for whatever reason, it's brought up that there is a writer on the team.

Let's be fair here: nowhere on the patreon page is it stated that there is, in fact, "a writer on the team". The only paragraph that mentions writing is the following:

>The game wouldn’t be half as fun without awesome music, character sprites, sound effects and writing. We want to give back to the awesome people who keep on delivering just that—and also bring new people on board.

Which is correct: the game does contain writing (currently yours), and we are willing to repay the contributors (currently just you), if they want to be repaid. The "keep on delivering" bit may be misleading, but blame @lachs0r for that :)

>I did this under the expectation that this game would remain free and no profit would be made off of it. Technically speaking it could be argued that this isn't profit but donating,

It isn't just a technicality, it's the hard truth. The game is and will always be [100% free](https://github.com/taisei-project/taisei/blob/master/COPYING) in every sense of the word. It does not cost anything; it has no ads or premium features; the source code is freely available for anyone to build, study, modify, use, and redistribute; it's developed completely in the open by volunteers and otherwise hits all the checkmarks of [Free and Open Source Software](https://en.wikipedia.org/wiki/Free_and_open-source_software).

And indeed, **no profit is being made off of it**. The intention of the Patreon is to cover a tiny fraction of the costs that go directly into developing Taisei, such as the sum we've collectively paid for @afensorm's art, or @lachs0r's website hosting expenses, or what we would've paid *you* if you were charging. None of it goes into our pockets. The "profit" we're making here is very much negative.

Actually, all of this is kind of a moot point, considering that our current Pateron balance is probably lower than the withdrawal fee.

>but at the very least I should not be mentioned on your Patreon at all because I feel it is dishonest.

You are not mentioned on our Patreon.

## TL;DR

I hope the wall of text helped clear some misunderstandings w.r.t. the purpose of the Patreon and where the hypothetical money goes.

In summary:

* If you'd still like to withdraw your writing, please confirm that and it'll be removed, no questions asked.
* If you'd like to receive a compensation for your work, please talk to me and @laochailan on IRC or Discord.
* If you'd like to come back to work on the game with us, whether volunteered or paid, welcome aboard. We could really use some help with the extra stage.
* I hope that you did not sockpuppet the other two conspicuously freshly made GitHub accounts, but if you did, please refrain from it in the future. While I won't hold it against you, it comes off as pretty immature.

**Comment by Lyricana** [comment]
I did not sockpuppet the other two accounts, they're just friends of mine that are a little upset about things as they stand.  I want to make that clear.

I guess in a sense I knew that even having a mention to the writing is inadvertently talking about me, because I do not see any new writer or new dialogue.  It made me upset and uncomfortable.

It's fine though.  I'd prefer the dialogue and everything related to be removed, thank you.  I didn't want to until you found a replacement for it, but it doesn't seem like it's naturally happening.  This was never a monetary issue for me; I never expected to be paid ever.  I just don't want to be associated with this project anymore.  I want to completely put it behind me and stay a player of it only.

**Comment by Akaricchi** [comment]
Should be fixed in 49dcd7c8d7521eef141f8374b816807fa8cf8f98 now. Please tell me if I missed anything.

Also please understand that we can not remove your contributions from the git history, or from the current and previous stable releases retroactively. The change has landed in the master development branch now, and will propagate to the next stable version (hopefully after we've replaced the stubs with actual story/dialogue).

I hope you and your friends are satisfied with this conclusion.

**Comment by Lyricana** [comment]
Thanks.  I am.
